### PR TITLE
Ergo42 biacco-biacco keymap

### PR DESCRIPTION
**Pull request by Biacco42**

### CODE REVIEW

File: keyboards/ergo42/keymaps/biacco-biacco/config.h [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "../../config.h"
```
Comment (drashna) [inline keyboards/ergo42/keymaps/biacco-biacco/config.h]: This should be removed, as it's not needed.

File: keyboards/ergo42/keymaps/biacco-biacco/keymap.c [hunk 2]
```diff
@@ -0,0 +1,161 @@
+#include "ergo42.h"
```
Comment (drashna) [inline keyboards/ergo42/keymaps/biacco-biacco/keymap.c]: Could you replace all of these includes with just `#include QMK_KEYBOARD_H`

Comment (Biacco42) [reply keyboards/ergo42/keymaps/biacco-biacco/keymap.c]: I will make another PR about that.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**